electronicparsers/openmx/parser.py

```diff
@@ -744,6 +816,19 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
             self.logger.warning('.md file does not contain enough MD steps')
 
         if mainfile_md_steps is not None:
+            if is_stdout_valid:
+                mainfile_uele_str = mainfile_md_steps[0].get('SCF')[0].get('Uele')
+                mainfile_uele_float = float(mainfile_uele_str)
+                mainfile_uele_precision = num_of_decimal_places(mainfile_uele_str)
+                desired_precision = min(mainfile_uele_precision, stdout_uele_precision)
```

Comment:
this is again my fault for not being exact enough when I said `so I think we should probably use something like abs(Uele1 - Uele2) < desired_precision to be safe, where desired precision would be based on the number of places for the less precise one.` Checking the precision at the runtime is probably overkill. I was thinking about just looking into the files how many places we have and than hardcoding a reasonable precision, which in this case could be something like 1e-8.

So yeah, just change the parsing back to floats, drop the is_close and num_of_decimal_places...